packages/test/test-end-to-end-tests/src/test/benchmark/opCriticalPath.time.spec.ts

```diff
@@ -6,27 +6,26 @@
 import { strict as assert } from "assert";
 
 import { ITestDataObject, describeCompat } from "@fluid-private/test-version-utils";
-import { benchmark } from "@fluid-tools/benchmark";
+import { benchmark, Phase, type BenchmarkTimingOptions } from "@fluid-tools/benchmark";
 import { IContainer } from "@fluidframework/container-definitions/internal";
-import {
-	ContainerRuntime,
-	DefaultSummaryConfiguration,
-} from "@fluidframework/container-runtime/internal";
+import { ContainerRuntime } from "@fluidframework/container-runtime/internal";
 import {
 	toIDeltaManagerFull,
 	ITestContainerConfig,
 	ITestObjectProvider,
 	timeoutPromise,
 } from "@fluidframework/test-utils/internal";
 
+export const ___x = Phase;
+
 const testContainerConfig: ITestContainerConfig = {
 	runtimeOptions: {
 		enableGroupedBatching: true,
+		chunkSizeInBytes: 1024 * 1024 * 1024,
```

Comment:
This is needed to avoid hanging... I think there must be a sneaky bug affecting the LoaderContainerTracker's pause/resume functionality, maybe some race condition exacerbated by larger queue size so just having one grouped batch in the queue avoids it?

Reply:
Actually this may not be needed now that we're making a new container for every benchmark batch